index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home page</title>
+  <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./styles.css">
+  <link rel="icon" href="./images/cap-1266204__340.webp">
+</head>
+
+<body>
+  <header>
+    <div class="contact-container">
+      <div></div>
+      <div class="contact">
+        <ul class="contact-ul">
+          <li><a href=""><i class="fa-brands fa-facebook"></i></a></li>
+          <li><a href=""><i class="fa-brands fa-twitter"></i></a></li>
+          <li class="contact-li"><a href="">English</a></li>
+          <li class="contact-li"><a href="">My Page</a></li>
+          <li class="contact-li"><a href="">sign Up</a></li>
+        </ul>
+      </div>
+    </div>
+    <div class="header-container">
+      <div class="header-container">
+        <div class="nav-container">
+          <div class="logo">
+            <div>
+              <ul>
+                <li>
+                  <i class="fa-solid fa-graduation-cap cap"></i>
+                  <a class="logo-link" href="#home.html">Educational Resources</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+
+          <div class="nav">
+            <ul class="nav-ul">
+              <li class="navigation" id="home.html"><a href="#">Home</a></li>
+              <li class="navigation"><a href="about.html">About</a></li>
+              <li class="navigation"><a href="#event">Event</a></li>
+              <li class="navigation"><a href="#guest">Guest</a></li>
+              <li class="navigation sponsor"><a href="#sponsor">Sponsor</a></li>
+            </ul>
+          </div>
```

Comment:
Nice job building the header section, it looks great👌 But let's improve the UX of the page by making it more semantic😉 Kindly  wrap every nav content in a `nav` tag, this makes our page more semantic and aids user accessibility😉